packs/data/agents-of-edgewatch-bestiary.db/siege-shard.json

```diff
@@ -668,7 +669,7 @@
                     "value": null
                 },
                 "description": {
-                    "value": "<p>Mental damage ignores a siege shard's hardness.</p>\n<p>Additionally, proving the siege shard's generative conflict is over with the Prove Peace activity reverts the creature to ordinary stone.</p>"
+                    "value": "<p>Mental damage ignores a siege shard's hardness.</p>\n<p>Additionally, proving the siege shard's generative conflict is over with the @UUID[Compendium.pf2e.actionspf2e.Ma93dpT4K7JbP9gu]{Prove Peace} activity reverts the creature to ordinary stone.</p>"
```

Comment:
We may need to circle back to this one, I think we've settled on not including one time adventure activities like this one. @SpartanCPA  thoughts?